addons/mail/models/mail_mail.py

```diff
@@ -39,7 +39,8 @@ class MailMail(models.Model):
     # recipients
     email_to = fields.Text('To', help='Message recipients (emails)')
     email_cc = fields.Char('Cc', help='Carbon copy message recipients')
-    recipient_ids = fields.Many2many('res.partner', string='To (Partners)')
+    recipient_ids = fields.Many2many('res.partner', string='To (Partners)',
+        context={'active_test': False})
```

Comment:
`mail.mail` records are not kept forever.
Maybe we should change the uid in the test.

Reply:
add a comment to explain why: if a partner has been deactivated (after the mail was sent), you should still see it as a recipient of the mail